modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/PaginationBarTag.java

```diff
@@ -186,25 +196,32 @@ protected int processStartTag() throws Exception {
 
 		JspWriter jspWriter = pageContext.getOut();
 
-		jspWriter.write("<div class=\"dropdown pagination-items-per-page\">");
-		jspWriter.write("<button class=\"dropdown-toggle btn btn-unstyled\"");
-		jspWriter.write(" type=\"button\">");
-		jspWriter.write(_activeDelta.toString());
-		jspWriter.write(StringPool.SPACE);
-
 		ResourceBundle resourceBundle = TagResourceBundleUtil.getResourceBundle(
 			pageContext);
 
-		jspWriter.write(
-			StringUtil.toLowerCase(LanguageUtil.get(resourceBundle, "items")));
+		if (_showDropDown) {
```

Comment:
I think if we go with this behavior by default in the taglib without maintaining a new API because in that case we don't need it since the taglib was just created, we could just use the `getPaginationBarDeltas()` property to check if we should render DropDown and avoid the use of `_showDropDown` here.

Reply:
Maybe ... as I said just above, I think that adding a new attribute on the tag and a corresponding attribute on the React component will let us  keep everything as it is now and also make it more simple.

Reply:
Yeah, I've been thinking about having this behavior already by default on the Taglib side, what do you think? For example, since we have an intermediate component between `ClayPaginationBarWithBasicItems` since it receives `deltas` with `null` or `undefined`, we can enable the API to not render `DropDown`. In that case, it would be safe to call `getPaginationBarDeltas()` here too.